### PR TITLE
Add an example encoding, for both presentation and wire formats

### LINKED ISSUE #8: Example
DNS RFCs always include the presentation form of something.  It's almost pointless here, but it might solidify things more.

### PR DESCRIPTION
**Pull request by davidben**
Fixes #8.

### CODE REVIEW

**Comment by davidben** [comment]
@martinthomson this look good to you?

File: draft-ietf-tls-key-share-prediction.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ The presentation `value` of the SvcParamValue is a comma-separated list ({{Appen
 
 The wire format of the SvcParamValue is a sequence of 2-octet numeric values in network byte order. An empty list of values is invalid.
 
+For example, a TLS server which prefers `x25519` (29) and also supports `secp256r1` (23) would a `tls-supported-groups` SvcParamValue containing 29 and 23. The presentation `value` would be "29,23". The wire format of the SvcParamValue would be four octets, represented in hexadecimal as `001d0017`.
+
```
Comment (martinthomson) [inline draft-ietf-tls-key-share-prediction.md:72]: This helps, but I was thinking something like this:

`````suggestion

``` dns
example.net.  7200  IN SVCB 3 server.example.net. (
    port="8004" tls-supported-groups=29,23 )
```
`````

(I'm now uncertain about that syntax though)

**Review by martinthomson — COMMENTED** [review]
This helps, but I was thinking

**Comment by davidben** [comment]
Incorporated your example with some minor fixups to the surrounding text.

**Review by bob-beck — APPROVED** [review]

**Pull request merged**